web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -99,3 +98,96 @@
     }
   }
 }
+
+.recipe--teaser {
+  display: flex;
+  flex-direction: column;
+  background-color: #f1f0f4;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+
+  .recipe-card-wrapper {
+
+    .teaser-image-container {
+      position: relative;
+
+      .highlighted-label {
```

Comment:
Set some left and right padding for this label so that text isn't touching the sides

Reply:
done